app/views/widgets/radioBoxHorizontal.scala.html

```diff
@@ -0,0 +1,13 @@
+@(field : play.api.data.Field, options: Seq[(String, String, Boolean)], optionTrigger: String = "", isCompound: Boolean = false)
+
+@import helpers.CSRSkinnyFieldConstructor._
+
+@createId(id:String, v:(String,String,Boolean))=@{id + "-"+v._1.replace(" ", "_").replace("/", "_")}
```

Comment:
I have a feeling I have seen this code before somewhere. It could be made simpler by passing only the value from v._1 and spacing it out a bit